app/controllers/admin/preview_controller.rb

```diff
@@ -35,4 +35,20 @@ def limit_attachment_access!
       forbidden!
     end
   end
+
+  def find_and_prepare_images(params)
+    images = Image.find(params.fetch(:image_ids, []))
+    images.map do |image|
+      {
+        id: image.image_data.carrierwave_image,
+        image_data_id: image.image_data_id,
+        edition_id: image.edition_id,
+        alt_text: image.alt_text,
+        url: image.url,
+        caption: image.caption,
+        created_at: image.created_at,
+        updated_at: image.updated_at,
+      }
+    end
+  end
```

Comment:
Do you think we should share the mapping function in the helper? So we don't have duplicate code?